### ISSUE TITLE
Finders are not working

### ISSUE THREAD

**Issue by moski** [siuying/NanoStoreInMotion#8]
Whenever i try to search/find for something, the app crashes with the following error:

LLVM ERROR: Program used external function 'rb_pointer_new2' which could not be resolved!

User.find(:name => 'moski')

**Comment by siuying** [comment]
Can you checkout lastest version of the project, `rake clean` it and run `rake spec` to see if there are any failure?

**Comment by moski** [comment]
It's failing with the following error:

2012-06-08 18:36:15.326 NanoStoreDemo[92031:f803] **\* Terminating app due to uncaught exception 'NameError', reason: 'uninitialized constant Term (NameError)

for some strange reason def all(_args)  is causing def find(_arg) .. for now, i am overriding the the "all" method

module NanoStore
  module FinderMethods
    def all(*args)
      # Legacy code ##
      search = NSFNanoSearch.searchWithStore(self.store)
      searchResults = search.searchObjectsWithReturnType(NSFReturnObjects, error:Pointer.new(:id))
      ## End legacy code ##

```
 if args[0].is_a?(Hash)
    sort_options = args[0][:sort] || {}
  else
    sort_options = {}
  end

  if sort_options.empty?
    self.store.objectsOfClassNamed(self.bare_class_name)
  else
    sort_descriptors = sort_descriptor_with_options(sort_options)
    self.store.objectsOfClassNamed(self.bare_class_name, usingSortDescriptors:sort_descriptors)
  end   
```

end

when i added the code inside ## Legacy code ## everything worked .. very strange

**Comment by siuying** [comment]
"'uninitialized constant Term (NameError)" this is cause the program cannot find motion-redgreen.

this might caused by a recent bug in bubble-wrap which change load path. I have updated the Gemfile such that it fixed at 0.3.1 (this should get corrected soon)

Please fetch latest code, bundle install it and try again.

**Comment by siuying** [comment]
I dont understand these

> for some strange reason def all(_args) is causing def find(_arg) 
> .. for now, i am overriding the the "all" method

But the spec is passing for me, and it should work on your side once all dependency stuff cleared!
